src/libs/SelectionScraper/index.js

```diff
@@ -105,6 +105,12 @@ const replaceNodes = (dom) => {
         if (htmlEngineUtils.isCommentTag(dom.attribs[tagAttribute])) {
             dom.children.push(new Element('br', {}));
         }
+    } else if (dom.name === 'div' && dom.children.length === 1 && dom.children[0].type !== 'text') {
+        // skip useless div because it's doing some issues on reconverting the HTML to markdown
+        // and we skip it only if:-
+        // 1- not contain specific attributes
+        // 2- have only one child and not a text
```

Comment:
i was thinking about this comment 

        // We are skipping divs that have only one child and that child is not a text node. This is because
        //these divs have no meaning for us and it's causing issues with reconverting the HTML to markdown.

Reply:
@aimane-chnaif I update the comment but delete the example above  I think we not need it.